lib/_http_outgoing.js

```diff
@@ -92,7 +92,7 @@ OutgoingMessage.prototype.setTimeout = function(msecs, callback) {
 
   if (callback) {
     if (typeof callback !== 'function')
-      throw new TypeError('callback must be a function');
+      throw new TypeError('"callback" must be a function');
```

Comment:
On the one hand, yes, they are complete sentences.

On the other hand, this is a typical usage:

    try {
        // whatever
    } catch (e) {
        console.error('Error "' + e.message +'" received while trying to foo.");
    }

I can go either way, but agree that whatever is decided, it should be consistent.